extensions/cache/deployment/src/test/java/io/quarkus/cache/test/runtime/ThrowExecutionExceptionCauseTest.java

```diff
@@ -55,74 +48,13 @@ public void testCheckedExceptionThrowDuringCacheComputation() {
     }
 
     @Test
-    public void testRuntimeExceptionThrowDuringCacheComputationWithLockTimeout() {
-        UnsupportedOperationException e = assertThrows(UnsupportedOperationException.class, () -> {
-            cachedService.throwRuntimeExceptionDuringCacheComputationWithLockTimeout();
+    public void testErrorThrowDuringCacheComputation() {
+        OutOfMemoryError e = assertThrows(OutOfMemoryError.class, () -> {
+            cachedService.throwErrorDuringCacheComputation();
         });
         assertEquals(FORCED_EXCEPTION_MESSAGE, e.getMessage());
     }
 
-    @Test
-    public void testBothLockTimeoutCodeBranches() throws InterruptedException {
```

Comment:
There used to be two code branches and we had to deal with asynchronous cache computations before, but today cache computations are done synchronously on the calling thread so this method has become completely useless.